### PR TITLE
Create ndzhwr.json

### PR DESCRIPTION
**Pull request by ndzhwr**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I just added my profile in `/data` directory -> `/data/ndzhwr.json`

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5001"></a>

### CODE REVIEW

File: data/ndzhwr.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    },
+      {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/_ndzhwr",
+      "icon": "FaInstagram"
+    },
+      {
+      "name": "Follow me on Github",
+      "url": "https://github.com/ndzhwr",
+      "icon": "FaGithub"
```
Comment (mahfuztbt) [inline data/ndzhwr.json:19]: delete `,` from line 20. only use `}`

**Review by shubhsharma19 — APPROVED** [review]
Looks good 🎉, however you can fix the indentation on line 11 and 16. :D

File: data/ndzhwr.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
```
Comment (Krishnapro) [inline data/ndzhwr.json:20]: Remove `,` after `}` in line 20
```diff
- },
+ }

**Review by Krishnapro — CHANGES_REQUESTED** [review]
Remove `,` after `}` in line 20

File: data/ndzhwr.json (hunk 2 above)
Comment (shubhsharma19) [reply data/ndzhwr.json:20]: Thanks for the mention 🎉

**Comment by ndzhwr** [comment]
I've removed the `,` @Krishnapro

**Review by shubhsharma19 — APPROVED** [review]
LGTM 🎉

**Review by samyabrata-maji — APPROVED** [review]

**Review by himanshu1221 — APPROVED** [review]
**LGTM :+1:**

**Review by mahfuztbt — APPROVED** [review]
it's  okay now

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**